core/clients/key_flow.go

```diff
@@ -97,13 +104,19 @@ func (c *KeyFlow) GetServiceAccountEmail() string {
 
 // GetToken returns the token field
 func (c *KeyFlow) GetToken() TokenResponseBody {
+	c.tokenMutex.RLock()
+	defer c.tokenMutex.RUnlock()
+
 	if c.token == nil {
 		return TokenResponseBody{}
 	}
+	// Returned struct is passed by value (because it's a struct: https://go.dev/play/p/0VQGLfP6vsg)
```

Comment:
interesting use of Go playground in a comment :laughing: 

do we really need it? maybe the just the rest of the comment is enough, to avoid noise in the code

Reply:
Well if you believe me, I guess not :wink:
Removed